origins_common/src/Plugin/Filter/CookieContentBlockerEmbedFilter.php

```diff
@@ -139,7 +139,15 @@ function ($matches) {
         $replacement = $matches[1];
         $entity = $this->entityRepository->loadEntityByUuid('media', $matches[2]);
 
-        if ($entity && $entity->bundle() === 'remote_video') {
+        // Don't add the cookie blocker wrapper if we are on the node edit page, as
+        // editors should always see the video preview.
+        $add_cookie_content_blocker = TRUE;
+        $this_route = \Drupal::routeMatch()->getRouteName();
+        if ($this_route == 'media.filter.preview') {
+          $add_cookie_content_blocker = FALSE;
+        }
+
+        if (($entity && $entity->bundle() === 'remote_video') && $add_cookie_content_blocker) {
```

Comment:
If you swapped the conditions here you could potentially avoid the entity check altogether. Or in other words: If `$add_cookie_content_blocker` is FALSE, then you know you don't need check anything more after that.

Reply:
Fair point, that's done now